src/NativeMessageHost/CMakeLists.txt

```diff
@@ -92,6 +93,7 @@ set_target_properties(${PROJECT_NAME} PROPERTIES
 
 if (FIREBREATH)
     link_boost_library(${PROJECT_NAME} filesystem)
+    link_boost_library(${PROJECT_NAME} regex)
```

Comment:
I don't know of any reason that this would be needed either; what prompted you to add the boost regex library? is something using it?